libtheora/VITABUILD

```diff
@@ -2,7 +2,7 @@ pkgname=libtheora
 pkgver=1.1.1
 pkgrel=1
 url="https://www.theora.org/"
-source=("http://downloads.xiph.org/releases/theora/${pkgname}-${pkgver}.tar.bz2")
+source=("http://downloads.xiph.org/releases/theora/$pkgname-$pkgver.tar.bz2")
```

Comment:
cool this pr.
thank you updating packages.
just one thing, could you restore braces to keep small patch if they didn't change?
(this file is one of cases)

Reply:
Done, let me know if I missed something.